### ISSUE TITLE
Cockpit 264.2 (RHEL 9.0) Cannot Check Software Updates on Fedora 36 Hosts (Unable to communicate with OSTree)

### ISSUE THREAD

**Issue by tonymartino** [cockpit-project/cockpit#17902]
Hello,
Connecting to Cockpit 264.2 running on up-to-date RHEL 9.0 will is not able to check for software updates on Fedora 36 host (Cockpit 278). When attempting to check for software updates on these hosts, Cockpit returns "Unable to communicate with OSTree" and "OSTree is not available on this system". OSTree is not a requirement for software updates on the platform.

**Where is the problem in Cockpit?**
Software Updates

**Server operating system**
RHEL

Server operating system version
9.0

What browsers are you using?
Chrome 107.0.5304.110 (Official Build) (64-bit)

System log
Nov 12 14:38:20 jello.inside.msnomer.com realmd[16422]: client gone away: :1.195
Nov 12 14:38:20 jello.inside.msnomer.com realmd[16422]: released daemon: :1.195
Nov 12 14:38:20 jello.inside.msnomer.com realmd[16422]: client gone away: :1.199
Nov 12 14:38:20 jello.inside.msnomer.com realmd[16422]: released daemon: :1.199
Nov 12 14:38:21 jello.inside.msnomer.com systemd[1]: rpm-ostreed.service - rpm-ostree System Management Daemon was skipped because of a failed condition check (ConditionPathExists=/ostree).
Nov 12 14:38:21 jello.inside.msnomer.com realmd[16422]: client using service: :1.208
Nov 12 14:38:21 jello.inside.msnomer.com realmd[16422]: holding daemon: :1.208

**Comment by garrett** [comment]
You shouldn't be using cockpit-ostree on a non-ostree system.

- RHEL 9 is not an ostree system
- Fedora Linux 36 is (usually) not an ostree-based system.

Systems based on rpm-ostree include:
- Fedora Silverblue
- Fedora Kinoite
- Fedora CoreOS
- Fedora IoT

If you aren't running Fedora's Silverblue, Kinoite, CoreOS, or IoT, you shouldn't have `cockpit-ostree` installed. Fedora Workstation and Fedora Server are _not_ based on rpm-ostree.

In other words, you can probably fix this problem on your machine if you run the following as root:

```
dnf remove cockpit-ostree
```

Closing this, as I'm pretty confident it will fix your problem. If removing the `cockpit-ostree` package does not fix this problem, please re-open this issue.

Either way, if you could report back with the results, that would be great! It'd be nice to know if this works for you or doesn't.

I hope this helps!
